src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditorContainer.js

```diff
@@ -85,7 +89,9 @@ class OrderTemplatesEditorContainer extends Component {
   };
 
   render() {
-    const { close, resources, match } = this.props;
+    const { close, resources, match, stripes } = this.props;
+    const formValues = getFormValues('orderTemplateForm')(stripes.store.getState()) || {};
```

Comment:
I don't think you need `|| {}` here. Basically, that could be a potential bug, you always have to pass `initialValues` to form, and in create case without pre-defined values it should be `{}`, so `getFormValues()` should always return some object.

Reply:
form is not create for the first render